src/minimega/router.go

```diff
@@ -46,6 +52,16 @@ type dhcp struct {
 	static map[string]string
 }
 
+type bgp struct {
+	bgpprocessname string
+	localip        string
```

Comment:
Go convention is [MixedCaps](https://golang.org/doc/effective_go.html#mixed-caps).

Reply:
resolved in latest commit